kubectl-plugin/waiops-status.md

```diff
@@ -259,5 +326,6 @@ chmod +x cp4waiops-samples/kubectl-plugin/kubectl-waiops
 cp cp4waiops-samples/kubectl-plugin/kubectl-waiops /usr/local/bin/kubectl-waiops
```

Comment:
If the user is on IBM cloud using some different terminal, the location of `oc` might be a bit different. 
Should we suggest something like running `type oc` to determine the folder the `oc` binary is in? Or does it not matter as long as it is executable?